perf/perf_producer.js

```diff
@@ -77,7 +77,7 @@ const Pulsar = require('../index.js');
   const numOfMessages = commander.messages;
   for (let i = 0; i < commander.iteration; i += 1) {
     const histogram = hdr.build({
-      bitBucketSize: 0,
+      bitBucketSize: 64,
```

Comment:
In hdr-histogram-js v1.1.2, `0` means `Float64Histogram`.
https://github.com/HdrHistogram/HdrHistogramJS/blob/ecd138c3fb58e5da9643771bc758c615c58af018/src/index.ts#L66-L78